website/docs/verifying-images.md

```diff
@@ -0,0 +1,260 @@
+# Verifying Release Images
+
+All official Dalec container images published to GitHub Container Registry (GHCR) are cryptographically signed and include attestations for build provenance. This enables users to verify that images are authentic and haven't been tampered with.
+
+## Why Verify Images?
+
+Image verification provides several security benefits:
+
+- **Authenticity**: Confirm the image was published by the official Dalec repository
+- **Integrity**: Detect if the image has been tampered with or modified
+- **Provenance**: Verify the image was built from the expected source code and workflow
+- **Supply Chain Security**: Comply with security best practices and policies (e.g., SLSA framework)
+
+## What Gets Signed
+
+The following container images are signed when published as releases:
+
+- **Frontend image**: `ghcr.io/project-dalec/dalec/frontend` (main BuildKit frontend)
+- **Worker images**: Various target-specific worker images (e.g., `ghcr.io/project-dalec/dalec/mariner2/container`)
```

Comment:
This is `mariner2/worker`, also maybe change the ref to something else (since mariner2 is EOL) like `noble/worker`